docs/reference/release-notes/6.6.asciidoc

```diff
@@ -59,4 +87,11 @@ Machine Learning::
 * Fix cause of "Sample out of bounds" error message. {ml-pull}335[#335]
 * Fix hang when closing a job or creating a forecast. This problem occurs if you
 created a forecast for a large job and temporary storage was not cleaned up.
-{ml-pull}352[#352] (issue: {ml-issue}350[#350])
\ No newline at end of file
+{ml-pull}352[#352] (issue: {ml-issue}350[#350])
+* Wait for autodetect to be ready in the datafeed {pull}37349[#37349]
+(issues: {issue}36810[#36810], {issue}37227[#37227])
+* Stop datafeeds when their jobs are stale {pull}37227[#37227]
+(issue: {issue}36810[#36810])
+* Fix x-pack usage regression caused by index migration {pull}36936[#36936]
```

Comment:
This one was broken and fixed during 6.6 development, so has never been a problem in a released build.  Therefore I'd leave it out.

Reply:
Ditto